strapi/helm/strapi/values.yaml

```diff
@@ -0,0 +1,95 @@
+# Default values for strapi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: gittysachin/strapi
```

Comment:
you should be able to reference this as `dkr.plural.sh/strapi/gittysachin/strapi` once the vendor images action runs.

Reply:
@michaeljguarino got it, so should I change the repository name to include `dkr.plural.sh/strapi` now?

Reply:
Changed the image url to `dkr.plural.sh`